### DISCUSSION TITLE
VanJS Form - Fully typed form state management library (with validation)

### DISCUSSION THREAD

**Discussion by kwameopareasiedu** [vanjs-org/van#219] [Show and tell]
![image](https://github.com/vanjs-org/van/assets/26204283/0361e319-9fa0-4b35-a235-9c3b3bda3747)

@Tao-VanJS Just published another plugin for the VanJS ecosystem. This is [**VanJs Form**](https://github.com/kwameopareasiedu/vanjs-form), which is fully types form state management library with validation (Currently supports only Yup. _Zod and Joi support coming in the near future_)

I hope it can help boost the uptake of the framework. It's so simple and intuitive, I love it.

Github: https://github.com/kwameopareasiedu/vanjs-form
NPM: https://www.npmjs.com/package/vanjs-form

**Comment by Tao-VanJS** [comment]
Thank you so much! I added your repo to community add-ons section.
